maintainers/maintainer-list.nix

```
@@ -15815,6 +15815,11 @@
     githubId = 358550;
     name = "Philip Lykke Carlsen";
   };
+  plebhash = {
+    email = "plebhash@proton.me";
+    github = "plebhash";
+    githubId = 147345153;
+  };
```

`name` must be defined. It's fine to just put your nym there like this.

```suggestion
  plebhash = {
    name = "plebhash";
    email = "plebhash@proton.me";
    github = "plebhash";
    githubId = 147345153;
  };
```